.pre-commit-config.yaml

```diff
@@ -0,0 +1,12 @@
+---
+default_stages: [commit]
+repos:
+  - repo: local
+    hooks:
+      - id: npm-groovy-lint
+        stages: [commit]
+        name: npm-groovy-lint
+        entry: bash -c 'npm-groovy-lint tests/jenkins src/jenkins vars --failon error'
```

Comment:
What does a failure look like? Will it won't even allow to commit the code or commit and then error out?

Reply:
It won't allow you to commit code. Errors only occur when an error is detected and this is set by the --failon flag.